jenkins-pipelines/rolling-upgrade-ami-arm.jenkinsfile

```diff
@@ -5,7 +5,8 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 rollingUpgradePipeline(
     backend: 'aws',
-    base_versions: '',  // auto mode
+    base_versions: '',  // auto mode,
+    linux_distro: 'centos',
```

Comment:
but this is incorrect...
our AMIs are ubuntu.

Reply:
the code calling to `./docker/env/hydra.sh get-scylla-base-versions --only-print-versions true --linux-distro null --scylla-repo` should be fixed, and have proper defaults.

Reply:
so, inside of the `get-scylla-base-versions` we need to get this `null` entry value, and fallback to its default (must be `ubuntu-focal`), other than adding to the `.jenkinsfile` the value